src/connector/C#/README.md

```diff
@@ -0,0 +1,440 @@
+## CSharp Connector
+
+* The C # connector supports: Linux 64/Windows x64/Windows x86.
+* C# connector can be download and include as normal table form [Nuget.org](https://www.nuget.org/packages/TDengine.Connector/).
+
+### Installation preparation
+
+* Install TDengine client, please refer to the[steps of installing connector driver](https://www.taosdata.com/en/documentation/connector#driver).
+* .NET interface file TDengineDrivercs.cs and reference sample TDengineTest.cs
+ are both located in the Windows client path:install_directory/examples/C#.
+* Install [.NET SDK](https://dotnet.microsoft.com/download)
+
+### Installation verification
+
+Run {client_installation_directory}/examples/C#/C#Checker/C#Checker.cs
+Need install .Net SDK first
```

Comment:
if this is new paragraph, finish previous paragraph with `.`, and add a break line.